src/utils/wormhole.ts

```diff
@@ -620,6 +620,12 @@ const WORMHOLE_CHAINS: any = {
     },
     getExplorerBaseURL: function ({ network = "Mainnet", value, base }: ExplorerBaseURLInput) {
       if (base === "address") return this.explorer?.[network] + "/address/" + value;
+      if (base === "token") {
+        if (value.length <= 6) {
```

Comment:
Assuming that all native token symbols on Noble contain 4 characters (USDC, EURe, USDN, etc), we can change this to 5 characters.

```suggestion
        if (value.length <= 5) {
```